web/app/js/components/PageHeader.jsx

```diff
@@ -27,7 +25,7 @@ export default class PageHeader extends React.Component {
             {!this.props.subHeaderTitle ? null : <div className="subsection-header">{this.props.subHeaderTitle}</div>}
             {!this.props.subHeader ? null : <h1>{this.props.subHeader}</h1>}
           </Col>
-          { this.props.hideButtons ? null :
+          {/* { this.props.hideButtons ? null :
```

Comment:
mind leaving a comment about disabling this?

Reply:
yeah good idea!